docs/topics/request-response.rst

```diff
@@ -200,12 +214,37 @@ Example::
         self.logger.info("Visited %s", response.url)
 
 In some cases you may be interested in passing arguments to those callback
-functions so you can receive the arguments later, in the second callback. You
-can use the :attr:`Request.meta` attribute for that.
+functions so you can receive the arguments later, in the second callback.
+The following example shows how to achieve this by using the
+:attr:`Request.cb_kwargs` attribute:
+
+::
+
+    # pass information to the next callback using the Request.cb_kwargs attribute
+    def parse(self, response):
+        request = scrapy.Request('http://www.example.com/index.html',
+                                 callback=self.parse_page2,
+                                 cb_kwargs=dict(main_url=response.url))
+        request.cb_kwargs['foo'] = 'bar'  # add more arguments for the callback
+        yield request
+
+    def parse_page2(self, response, main_url, foo):
+        yield dict(
+            main_url=main_url,
+            other_url=response.url,
+            foo=foo,
+        )
 
-Here's an example of how to pass an item using this mechanism, to populate
-different fields from different pages::
+.. caution:: :attr:`Request.cb_kwargs` was introduced in version ``1.7``.
```

Comment:
I’m thinking that we could just leave the warning about the version where cb_kwargs was introduced. If users realize that they are not using Scrapy 1.7, they should switch to the right version of the documentation anyway.

Reply:
True, I forgot about the possibility of navigating older versions, thanks for the suggestion.